cfpb/consumerfinance.gov
# Updates rural-or-underserved-tool data model + 2021 updates

Now 6-7x more efficient!

Testing:
 - Pull and run `yarn run gulp scripts:apps`
 - visit localhost:8000/rural-or-underserved-tool/
 - 2021 should be selected and functional
 - All other years should operate identically to https://www.consumerfinance.gov/rural-or-underserved-tool/

Once this is merged + released + deployed, we can delete the existing rout files from S3 (the new data is in a more idiomatic place, which also enables testing this PR!)

Also, this now automatically updates the listed years in the tool based on the current year, so future updates will only be putting data on S3.
